features/create_articles.feature

```diff
@@ -4,11 +4,15 @@ Feature: Create articles
   I would like to be able to create articles
 
   Background:
-    Given I visit the Homepage
+    Given the following user exists
+      | email           | password    | password_confirmation |
+      | harald@norge.no | OsloOslo123 | OsloOslo123           |
+    And I am logged in as "harald@norge.no"
```

Comment:
Now that you have Active Admin, I assume that the article creation process will be moved to that interface, right?